src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -68,55 +68,59 @@
 					fill-color="#ffffff"
 					decorative />
 			</button>
-			<button
+			<Actions
 				v-if="!screenSharingButtonHidden"
 				id="screensharing-button"
 				v-tooltip="screenSharingButtonTooltip"
 				:aria-label="screenSharingButtonAriaLabel"
 				:class="screenSharingButtonClass"
 				class="app-navigation-entry-utils-menu-button"
-				@click.stop="toggleScreenSharingMenu">
+				boundaries-element="boundaryElement"
```

Comment:
you'll also need `container="#content-vue"` to make it work full screen, see other actions